contrib/packaging/rpm/cfg/cilium.spec.envsubst

```diff
@@ -0,0 +1,70 @@
+%global commit0 ${COMMIT}
+%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
+
+Name:          cilium
+Version:       ${VERSION}
+Release:       0.git%{shortcommit0}%{?dist}
+Summary:       BPF & XDP for containers
+License:       Apache
+URL:           https://github/cilium/cilium
+ExclusiveArch: x86_64
+Source0:       https://github.com/cilium/%{name}/archive/%{commit0}.tar.gz#/%{name}-%{shortcommit0}.tar.gz
+BuildRequires: golang < 1.8.0, glibc-devel(x86-32)
+Requires:      docker-engine >= 1.12, clang >= 3.8, glibc-devel(x86-32), iproute >= 4.6, kernel >= 4.8
+
+%description
+Cilium provides fast in-kernel networking and security policy enforcement
+for containers based on eBPF programs generated on the fly. It is an
+experimental project aiming at enabling emerging kernel technologies such
+as BPF and XDP for containers.
+
+%pre
+getent group cilium >/dev/null || groupadd -f -r cilium
+exit 0
+
+%prep
+mkdir -p %{name}-%{commit0}/src/github.com/cilium/%{name}
+tar zxf %{SOURCE0} --strip-components=1 -C %{name}-%{commit0}/src/github.com/cilium/%{name}
+
+%build
+export GOPATH=$(pwd)/%{name}-%{commit0}
+export PKG_BUILD=1
+cd %{name}-%{commit0}/src/github.com/cilium/%{name}
+echo "%{version}-0.git%{shortcommit0}" > VERSION
+make
+
+%install
+export GOPATH=$(pwd)/%{name}-%{commit0}
+export PKG_BUILD=1
+export DESTDIR=%{buildroot}
+cd %{name}-%{commit0}/src/github.com/cilium/%{name}
+%make_install
+
+mkdir -p "%{buildroot}/%{_libdir}/systemd/system"
+mkdir -p "%{buildroot}/etc/sysconfig"
+
+cp contrib/systemd/*.service "%{buildroot}/%{_libdir}/systemd/system"
+chmod 644 %{buildroot}/%{_libdir}/systemd/system/*
+
+cp contrib/systemd/cilium "%{buildroot}/etc/sysconfig"
+chmod 644 "%{buildroot}/etc/sysconfig/cilium"
+
+%files
+/etc/bash_completion.d/cilium
```

Comment:
Out of curiosity, isn't this lacking the %{buildroot}? -> `%{buildroot}/etc/bash_completion.d/cilium`